libs/cart/testing/src/factories/cart-item.factory.ts

```diff
@@ -14,14 +14,15 @@ export class MockCartItem implements DaffCartItem {
   qty = faker.random.number({min:1, max:100});
   price = faker.random.number(1500);
   row_total = this.qty * this.price
+  total_discount = faker.random.number(1000);
```

Comment:
I think we should guarantee that this will be less than the price field

Reply:
Changed